Agents.md

```
@@ -0,0 +1,201 @@
+yarn install --immutable
+```
+
+- `deps.cirru` 的 `:calcit-version` 与 `package.json` 的 `@calcit/procs` 保持同版本链路。
+
+---
+
+## TypeScript / WGSL 层
+
+### 文件结构
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)